kratos/python_scripts/new_linear_solver_factory.py

```diff
@@ -68,6 +68,20 @@ def ConstructSolver(configuration):
     elif (solver_type == "ParallelMKLPardisoSolver"):
         import KratosMultiphysics.MKLSolversApplication
         linear_solver = KratosMultiphysics.MKLSolversApplication.ParallelMKLPardisoSolver(configuration)
+    
+    ################################## following solvers need importing the EigenSolversApplication
+    elif(solver_type == "Eigen_SparseLU"):
```

Comment:
I know that this factory will be moved to c++ soon, but still the comment on using the [Style Guide](https://github.com/KratosMultiphysics/Kratos/wiki/Style-Guide#json-configuration-file) for the names of the solver

Reply:
Did not saw this but I think "**solver**_type": "Eigen_SparseLU**Solver**" is a bit redundant.

Reply:
I meant `solver_type = eigen_sparse_lu`